web/js/blocks/cw-event.js

```diff
@@ -14,61 +14,88 @@
  * limitations under the License.
  */
 
-'use strict';
+'use strict'
```

Comment:
I really don't want all the semicolons deleted from JavaScript code.

Reply:
Could you explain why, in a nutshell? I've been writing JS code without semicolons for several years now without running into any issues - although I have to admit that there was always a bundler involved.

Reply:
Sure.  It comes down to a few things:

1. Almost all JavaScript I've seen elsewhere uses semicolons, and there's value in doing things the way most others do them.  It makes the code a little easier for new programmers to work with.  I've worked at Facebook and Google, and both of their style guides require them.  I just looked around for polls and surveys and such, and consistently about 90% of JavaScript code and 90% of people report using explicit semicolons.
2. Automatic semicolon insertion is complex, and I believe syntax should be simple and predictable.  I worry about syntax mistakes that are turned into valid code by automatically adding semicolons, and I'd prefer to let ESLint clarify what's going on by adding the semicolon there.
3. Though rare, there have even been cases where changes to JavaScript in browsers have broken older code because new syntax prevented a semicolon from being automatically added.

But, mostly it's the first one.  If most of the JavaScript people see has semicolons, removing them just makes this code base harder to work in.

Reply:
Got it. I don't care too much about this. For what it's worth, my biggest pet peeve is excessive indentation - I'd really like to keep nesting to a minimum.